packages/shared/src/cachedFetch.ts

```diff
@@ -1,3 +1,5 @@
+import { fetch } from "undici";
```

Comment:
This seems _very easy_ to forget to do. Can we add a Lint rule to disallow using the global `fetch`? Else I can see us reintroducing this problem in the future.

Reply:
PRO-755 , it should be a super quick ticket but I'm wrapping up for today and I don't have time to switch branches and test it out. I'm also unsure what's the current state of lining in this repo

Reply:
🫡